### ISSUE TITLE
secrets manager: leading dashes in secret-string are interpreted as arguments

### ISSUE THREAD

**Issue by madsem** [aws/aws-cli#4318]
Referencing #1135

```shell
set region = $argv[2]
        or set region $AWS_DEFAULT_REGION

        set key (cat $argv[1])

        aws secretsmanager \
        create-secret --name "$argv[1]" \
        --description "SSH key" \
        --secret-string "$key" \
        --region $region
```

`aws: error: argument --secret-string: expected one argument`

**Comment by justnance** [comment]
@madsem - Thank you for opening a new issue and referencing closed issue #1135.  It appears the current status of related [Python bug](https://bugs.python.org/issue9334) is still pending under `patch review`.  From the information you have provided, it appears the error is returning because the value of `--secret-string "$key"` may begin with a dash "-".  If this is correct, the quickest work around would be to replace the "$key" value with one that does not begin with a dash. 

Please let us know if this helps.

**Comment by iftheshoefritz** [comment]
You can add `--param=-value-with-leading-minus`
